File: src/main/java/com/paragon/client/systems/module/impl/movement/ReverseStep.java
package com.paragon.client.systems.module.impl.movement;

import com.paragon.client.systems.module.Module;
import com.paragon.client.systems.module.Category;

/**
 * @author Wolfsurge
 */
public class ReverseStep extends Module {

    public ReverseStep() {
        super("ReverseStep", Category.MOVEMENT, "Moves you down when you walk off of a block");
    }

    @Override
    public void onTick() {
        if (nullCheck()) {
            return;
        }

        // Check that we want to fall
        if (mc.player.onGround && !mc.player.isInWater() && !mc.player.isInLava() && !mc.player.isOnLadder() && !mc.gameSettings.keyBindJump.isKeyDown()) {
            mc.player.motionY = -10.0;
        }
    }
}

File: src/main/kotlin/com/paragon/client/systems/module/impl/movement/ReverseStep.kt
package com.paragon.client.systems.module.impl.movement

import com.paragon.api.module.Category
import com.paragon.api.module.Module

/**
 * @author Surge, SooStrator1136
 */
object ReverseStep : Module("ReverseStep", Category.MOVEMENT, "Moves you down when you walk off of a block") {

    override fun onTick() {
        minecraft?.player?.let { player ->
            // Check that we want to fall
            if (player.onGround && !player.isInWater && !player.isInLava && !player.isOnLadder && !minecraft.gameSettings.keyBindJump.isKeyDown) {
                minecraft.player.motionY = -1.0 //shouldnt be 10
            }
        }
    }
//i dont like java this shit is pure aids
}